README.md

```diff
@@ -26,6 +26,28 @@ under that directory will contain instructions on how to run the sample.
 
 If desired, you can easily use the Maven Archetype from a Maven capable IDE such as [Eclipse](https://www.eclipse.org/ide). The generated starter code is simply Maven projects. You can easily load, explore and run the code in a Maven capable IDE such as [Eclipse](https://www.eclipse.org/ide).
 
+## Instructions On How To Use Archetypes From An IDE Such As Eclipse
+
+# If you are using an enterprise version of Eclipse for a Jakarta EE project, you can follow these steps to create a new Maven project with an archetype:
+
+Open Eclipse and select "File > New > Other..." from the menu bar.
```

Comment:
Selecting "Other" is unnecessary. Maven has been a top level choice in Eclipse IDE for some time now.